app/Http/Controllers/UserController.php

```diff
@@ -17,19 +18,22 @@ public function get(User $user)
 
     public function store(StoreUserRequest $request)
     {
-        $inviteKeyId = $request->get('invite_key');
+        $inviteKeyValue = $request->get('invite_key');
```

Comment:
Nog een klein detail. Voortaan graag `$request->invite_key` ipv hoe je het nu doet. Is vooral belangrijk om de code style consistent te houden.

Niet alleen hier maar op elke andere plek.